Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+                file.write('    '.join(map(str, [999., 999.])) + '\n')
+
+                # Write duct profile coordinates, using a tab delimiter
+                for row in xy_duct:
+                    file.write('    '.join(map(str, row)) + '\n')
+
+            return None
+
+
+    class fileHandlingMTFLO:
+
+        def __init__(self, *args) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)